tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -183,6 +185,41 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 		return restConfig, storageclass, profileID
 	}
 
+	staticProvisioningPreSetUpUtilForVMDKTests := func(ctx context.Context) (*restclient.Config, *storagev1.StorageClass, string) {
```

Comment:
does this have to be a func pointer?

Reply:
No, to avoid duplicate code on vmdk test cases I created this as separate func. This does only few pre-setups for VMDK tests to run.